source/val/validate_image.cpp

```diff
@@ -2249,12 +2322,106 @@ spv_result_t ImagePass(ValidationState_t& _, const Instruction* inst) {
     case spv::Op::OpImageSparseTexelsResident:
       return ValidateImageSparseTexelsResident(_, inst);
 
+    case spv::Op::OpImageSampleWeightedQCOM:
+    case spv::Op::OpImageBoxFilterQCOM:
+    case spv::Op::OpImageBlockMatchSSDQCOM:
+    case spv::Op::OpImageBlockMatchSADQCOM:
+      return ValidateImageProcessingQCOM(_, inst);
+
     default:
       break;
   }
 
   return SPV_SUCCESS;
 }
 
+bool IsImageInstruction(const spv::Op opcode) {
```

Comment:
Is this not something we have anyway of generating currently? Do you really care for `OpImageSparseTexelsResident` which doesn't have a `OpImage` ID to it?

Reply:
Thanks for the comment. I was not very sure about OpImageSparseTexelsResident, so I just added it. Let me remove it.